core-bundle/contao/dca/tl_content.php

```diff
@@ -810,6 +811,20 @@
 			'inputType'               => 'text',
 			'eval'                    => array('rgxp'=>'datim', 'datepicker'=>true, 'tl_class'=>'w50 wizard'),
 			'sql'                     => "varchar(10) COLLATE ascii_bin NOT NULL default ''"
+		),
+		'reg_close' => array
+		(
+			'inputType'               => 'select',
+			'options'                 => array('close_deactivate', 'close_delete'),
+			'eval'                    => array('tl_class'=>'w50'),
+			'reference'               => &$GLOBALS['TL_LANG']['tl_content'],
+			'sql'                     => "varchar(32) COLLATE ascii_bin NOT NULL default ''"
+		),
+		'reg_deleteDir' => array
+		(
+			'inputType'               => 'checkbox',
+			'eval'                    => array('tl_class'=>'w50'),
+			'sql'                     => array('type' => 'boolean', 'default' => false),
```

Comment:
@contao/developers should I wait for https://github.com/contao/contao/pull/8838 to be merged first?